samples/durable-task-sdks/dotnet/Agents/PromptChaining/AgentChainingSample.sln

```diff
@@ -0,0 +1,33 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
For new projects, we should consider using the [new slnx format](https://devblogs.microsoft.com/dotnet/introducing-slnx-support-dotnet-cli/), which is much simpler and easier to maintain.

Reply:
Note that this requires use of .NET 9+ tooling (if not runtime).